tokens/contracts/erc-1155/ERC1155Base.sol

```diff
@@ -39,5 +40,9 @@ abstract contract ERC1155Base is OwnableUpgradeable, ERC1155DefaultApproval, ERC
         return super.supportsInterface(interfaceId);
     }
 
+    function mintAndTransfer(LibERC1155LazyMint.Mint1155Data memory data, address to, uint256 _amount) public override validateMinter {
```

Comment:
Does it make sense to inherit the `MintControl` directly inside the `ERC1155Lazy` so we don't need to override the function and we can update a single `mintAndTransfer` function?

This way also other contracts that we might have missed would benefit from this change as well (for example the ERC721RaribleUser contract that is not covered here)